src/plugins/data/common/search/search_source/types.ts

```diff
@@ -112,10 +115,16 @@ export interface SearchSourceFields {
    * {@link IndexPatternService}
    */
   index?: DataView;
-  searchAfter?: EsQuerySearchAfter;
   timeout?: string;
   terminate_after?: number;
 
+  searchAfter?: estypes.SortResults;
+  /**
+   * Allow querying to use a point-in-time ID for paging results
+   * Requires searchAfter when the page index is > 1.
```

Comment:
This should be enforced in `search source`, before setting the pit.

Reply:
we chatted a bit to clarify this, and agreed it's better to just get rid of this comment.